### PR TITLE
Fix styling: inconsistent shadow of result bubble

### PR DESCRIPTION
**Pull request by lgeiger**

### CODE REVIEW

File: styles/hydrogen.less [hunk 1]
```diff
@@ -96,6 +96,8 @@
     .bubble-output-container {
         color: lighten(@syntax-text-color, 10%);
         background-color: @bubble-background;
+        // in order to don't accept the shadow from action-panel
```
Comment (n-riesco) [inline styles/hydrogen.less]: accept?

Comment (lgeiger) [inline styles/hydrogen.less]: Oh sorry my english isn't that great. I meant: `in order to be above the shadow of action panel`

**Comment by n-riesco** [comment]
I'm impressed with this attention to detail!

**Comment by lgeiger** [comment]
I noticed it and was bored during a long train ride  today :D

**Pull request merged**